packages/plugins/i18n/admin/src/contentManagerHooks/mutateEditViewLayout.js

```diff
@@ -5,28 +5,6 @@ import StrikedWorld from '@strapi/icons/EarthStriked';
 import LabelAction from '../components/LabelAction';
 import { getTrad } from '../utils';
 
-const enhanceRelationLayout = (layout, locale) =>
-  layout.map((current) => {
-    const labelActionProps = {
-      title: {
-        id: getTrad('Field.localized'),
-        defaultMessage: 'This value is unique for the selected locale',
-      },
-      icon: <I18N aria-hidden />,
```

Comment:
Same here I think it should be handled the plugin way instead of hardcoding this in the CM because it goes against the plugin idea where plugin A opens doors for any plugins, as if one plugin can do it then all plugins can also do it.

Reply:
I haven't checked that part yet, but I thought this is being handled by the `labelAction` passed to the label automatically?

Reply:
You are right but it is injected by i18n on the i18n code side not in the CM.

Reply:
I understand now :) Fix is the same as above: https://github.com/strapi/strapi/pull/14555